src/Tests/Microsoft.NET.ToolPack.Tests/GivenThatWeWantToPackAToolProject.cs

```diff
@@ -125,10 +126,10 @@ public void It_contains_runtimeconfig_for_each_tfm(bool multiTarget)
             }
         }
 
-        [Theory]
+        [WindowsOnlyTheory]
         [InlineData(true)]
         [InlineData(false)]
-        public void It_does_not_contain_apphost_exe(bool multiTarget)
+        public void Given_Windows_It_does_not_contain_apphost_exe_but_RunCommand_is_apphost_exe(bool multiTarget)
```

Comment:
I'm confused about the change to the tests.

Shouldn't it be the apphost for the `RunCommand` in both cases still be the apphost?  I see nothing that would limit it to just Windows from these changes.

I don't see why a change to pack would impact this test, other than `PackAsTool` projects should now debug/run with the apphost since they'll set `UseAppHost` by default.

Reply:
could you look at it again? I combined 2 tests

Reply:
Looks good.  I think we can probably change the test name back.

Reply:
Changed the name back